frontend/src/views/Layout/DashboardLayout_gdd.vue

```diff
@@ -78,10 +84,24 @@ export default {
       this.$store.dispatch('logout')
       this.$router.push('/login')
     },
+    async loadBalance() {
+      const result = await communityAPI.balance(this.$store.state.session_id)
+      if (result.success) {
+        this.balance = result.result.data.balance / 10000
+      } else {
+        // what to do when loading balance fails?
```

Comment:
toast + 0 ? toast + NaN/NA?

Reply:
balance is 0 by default. For now I would like to assume that this cannot fail, and touch it again, when we have a proper toaster.

Reply:
Issue pls